### PR TITLE
[ dress code ] Keybase GPG GitHub

### PR DESCRIPTION
**Pull request by rogeruiz**
Setting up GPG for GitHub using Keybase. Inspired by https://github.com/pstadler/keybase-gpg-github.

### CODE REVIEW

**Comment by rogeruiz** [comment]
Before this gets merged in, I should include a link to this helpful resource from @ahmadnassri https://www.ahmadnassri.com/blog/github-gpg-keybase-pgp/

It's really helpful in debugging some of the identity issues around email addresses in Keybase.

**Comment by ahmadnassri** [comment]
:+1: happy to help / answer questions if needed

File: dress-code/_posts/2017-02-04-setting-up-keybase-gpg-github.markdown [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+```sh
+
+keybase pgp list
+
+
+Keybase Key ID: $KEY_ID_FOR_QUERY
+PGP Fingerprint: MYP GP FIN GER PR INT
+PGP Identities:
+   Name Last <hello@example.com>
```
Comment (rogeruiz) [inline dress-code/_posts/2017-02-04-setting-up-keybase-gpg-github.markdown]: Mention that your email needs to exist in the list and in GitHub for the verified commit / tags to work. If the email address is one that doesn't exist or isn't used by Git, the PGP work won't give you the green verified icon

Comment (rogeruiz) [inline dress-code/_posts/2017-02-04-setting-up-keybase-gpg-github.markdown]: Added a whole section here on it. https://github.com/rogeruiz/rogeruiz.github.io/pull/8/files#diff-bb8c94be2d4a66a9be062fcb23f1f9aaR140

**Pull request merged**